tests/network/sriov/conftest.py

```diff
@@ -220,6 +220,7 @@ def restarted_sriov_vm4(request, sriov_vm4):
     LOGGER.info(f"Reboot number {request.param}")
     sriov_vm4.restart(wait=True)
     sriov_vm4.wait_for_agent_connected()
+    wait_for_vm_interfaces(vmi=sriov_vm4.vmi)
```

Comment:
I think this fits in the test body itself, but considering I think this whole test package needs to be refactored, it can be considered a nit.